### PR TITLE
feat: add image preview to info panel PE-1368

### PR DESCRIPTION
**Pull request by jdaev**
Adds image previews to the info panel with a cache.

### CODE REVIEW

File: lib/blocs/fs_entry_preview/fs_entry_preview_cubit.dart [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  final SelectedItem? maybeSelectedItem;
+
+  final DriveDao _driveDao;
+  final AppConfig _config;
+  final ArweaveService _arweave;
+  final ProfileCubit _profileCubit;
+
+  StreamSubscription? _entrySubscription;
+
+  final previewMaxFileSize = 1024 * 1024 * 100;
```
Comment (matibat) [inline lib/blocs/fs_entry_preview/fs_entry_preview_cubit.dart:28]: 100 MiB?

File: lib/blocs/fs_entry_preview/fs_entry_preview_cubit.dart [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+                default:
+                  emit(FsEntryPreviewUnavailable());
+              }
+            } else {
+              emit(FsEntryPreviewUnavailable());
+            }
+          });
+          break;
+
+        default:
```
Comment (matibat) [inline lib/blocs/fs_entry_preview/fs_entry_preview_cubit.dart:87]: Empty default case

File: lib/l10n/app_en.arb [hunk 3]
```diff
@@ -323,6 +323,10 @@
   "@uploadDragAndDrop": {
     "description": "The action of uploading a file"
   },
+  "itemPreviewEmphasized": "PREVIEW",
```
Comment (matibat) [inline lib/l10n/app_en.arb:326]: We need Spanish as well: #466

File: pubspec.lock [hunk 4]
```diff
@@ -711,7 +711,7 @@ packages:
       name: plugin_platform_interface
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.0.2"
+    version: "2.1.2"
```
Comment (matibat) [inline pubspec.lock:714]: Is this intended? I don't see any change in the `pubspec.yaml` :thinking:

**Review by matibat — COMMENTED** [review]
It's looking good. Left some comments

File: lib/blocs/fs_entry_preview/fs_entry_preview_cubit.dart (hunk 1 above)
Comment (jdaev) [reply lib/blocs/fs_entry_preview/fs_entry_preview_cubit.dart:28]: Yeah, @vilenarios requested it.

File: lib/l10n/app_en.arb (hunk 3 above)
Comment (jdaev) [reply lib/l10n/app_en.arb:326]: Gracias

**Review by thiagocarvalhodev — DISMISSED** [review]
Just left some non-block comments!

File: lib/blocs/fs_entry_preview/fs_entry_preview_cubit.dart (hunk 2 above)
Comment (jdaev) [reply lib/blocs/fs_entry_preview/fs_entry_preview_cubit.dart]: [Updated](https://github.com/ardriveapp/ardrive-web/pull/462/commits/81e3656f4ffe94cd701992eba436e4583b149c0d)

**Review by fedellen — DISMISSED** [review]
Looking good, thanks @jdaev!

**Review by fedellen — APPROVED** [review]

**Pull request merged**